addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -964,10 +964,10 @@ const Many2OneAvatar = FieldMany2One.extend({
         this._super.apply(this, arguments);
         if (this.mode === 'readonly') {
             this.template = null;
-            this.tagName = 'div';
             this.className = 'o_field_many2one_avatar';
-            // disable the redirection to the related record on click, in readonly
-            this.noOpen = true;
+            // by default, disable the redirection to the related record on click, in readonly
+            this.noOpen = 'noOpen' in this.nodeOptions ? this.nodeOptions.noOpen : true;
```

Comment:
Define the option as `no_open` instead of `noOpen`, to be consistent with the rest of the widget options :-)
Don't forget to adapt it to the widget as well.

Reply:
Done ;-)